compiler/AST/primitive.cpp

```diff
@@ -872,6 +872,9 @@ initPrimitive() {
   // for that loop launches.
   prim_def(PRIM_GPU_SET_BLOCKSIZE, "gpu set blockSize", returnInfoVoid, true);
 
+  // Generates call that produces runtime error when run by a GPU
```

Comment:
produces runtime error when **not** run by a GPU

Reply:
Good catch, fixed it.